android/java/org/chromium/chrome/browser/appmenu/BraveShieldsMenuAdapter.java

```diff
@@ -725,7 +726,7 @@ public void onAnimationStart(Animator animation) {
 
     static class StandardMenuItemViewHolder {
         public TextView text;
-        public AppMenuItemIcon image;
+        public /*AppMenuItemIcon*/ChromeImageView image;
```

Comment:
what is the comment here for?

Reply:
These are commented out just to make it build. There's a follow up issue for the Android team to fix:
https://github.com/brave/brave-browser/issues/7726